tests/test_stream_daq.py

```diff
@@ -104,7 +104,7 @@ def test_csv_output(tmp_path, default_streamdaq, write_metadata, caplog):
         # actually not sure what we should be looking for here, for now we just check for shape
         # this should be the same as long as the test data stays the same,
         # but it's a pretty weak test.
-        assert df.shape == (909, 11)
+        assert df.shape == (910, 11)
```

Comment:
this test is not great anyway, would be better to test some known values (or otherwise validate the correctness of the csv)

Reply:
Yes, that's in my backlog and will do it soon. More details in the comment below.